src/pages/patientView/SampleManager.tsx

```diff
@@ -197,6 +197,16 @@ class SampleManager {
         }
         return "";
     }
+
+    mapComponentForSample(sampleId: string, sampleIdToComponentMap: {[s:string]:JSX.Element | undefined}) {
```

Comment:
I don't see this method is used in other places. Are you planing to write tests for it, if not, probably better just merge with `getSampleIdToIconComponentMap`?

Reply:
dropped both these functions, think they were leftover from initial dev